### PR TITLE
replaced nginx by security article, which also covers:

### PR DESCRIPTION
**Pull request by kaikreuzer**
- changing HTTP(S) port
- Jetty SSL certificate
- restriction of network interfaces to use
- options for remote access (VPN, myopenHAB, reverse proxy)

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Fixes https://github.com/openhab/openhab-docs/issues/175

File: installation/security.md [hunk 1]
```diff
@@ -1,30 +1,78 @@
...
 
-# Running openHAB Behind a Reverse Proxy
+# Securing access to openHAB
+
+openHAB has mainly two ways to be accessed:
+
+1. Through the command line console, which is done through ssh and thus always authenticated and encrypted. You will find all details about this in the [Console documentation](/administration/console).
+2. Through HTTP(S), which we will look at in the following.
+
+{::options toc_levels="2..3"/}
```
Comment (ThomDietrich) [inline installation/security.md]: 2..3 by choice? adding 4 might be a good idea

File: installation/security.md [hunk 2]
```diff
@@ -1,30 +1,78 @@
...
+* TOC
+{:toc}
+
+## Encrypted Communication
+
+### Webserver Ports
+
+openHAB has a built-in webserver, which listens on port 8080 for HTTP and 8443 for HTTPS requests. In general, it is advised to use HTTPS communication over HTTP.
+
+The default ports 8080 and 8443 can be changed by setting the environment variables `OPENHAB_HTTP_PORT` resp. `OPENHAB_HTTPS_PORT`. In an apt installation, you would best do this in the file `/etc/defaults/openhab2`.
```
Comment (ThomDietrich) [inline installation/security.md]: Feels like after the second sentence something addressing manual setups is missing.

File: installation/security.md [hunk 3]
```diff
@@ -1,30 +1,78 @@
...
 Running openHAB behind a reverse proxy allows you to access your openHAB runtime via port 80 (HTTP) and 443 (HTTPS).
 It also provides you a simple way of **protecting your server** with authentication and secure certificates.
 
-This page is structured as follows:
-
-{::options toc_levels="2..4"/}
-
-* TOC
-{:toc}
+The good news is that [openHABian](openhabian) already comes with a preconfigured NGINX that can be used as a reverse proxy. For all others, please see the following chapters for a detailed description on how such a setup can be achieved.
```
Comment (ThomDietrich) [inline installation/security.md]: Nice :D although, it's just an option the user still has to select. 

```
The good news is that [openHABian]({{base}}/installation/openhabian.html) already offers the possibility to activate a preconfigured NGINX reverse proxy, which includes setting up authentication and a valid [Let's Encrypt](https://letsencrypt.org) certificate.
```

File: installation/security.md (hunk 1 above)
Comment (kaikreuzer) [inline installation/security.md]: by intention. Adding level 4 makes the TOC huge (and all the NGINX details are not relevant for most users). I tried to add a 4..5-toc below in the NGINX section, but unfortunately, you can only have one toc per page.

File: installation/security.md (hunk 2 above)
Comment (kaikreuzer) [inline installation/security.md]: No, in a manual setup you have to do it by "setting the environment variables `OPENHAB_HTTP_PORT` resp. `OPENHAB_HTTPS_PORT`" - how you do this, depends on your OS and installation type, so I didn't say anything further.

File: installation/security.md (hunk 3 above)
Comment (kaikreuzer) [inline installation/security.md]: changed

**Pull request merged**